internal/plugin/provider.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/aiven/terraform-provider-aiven/internal/plugin/service/organization/address"
 	"github.com/aiven/terraform-provider-aiven/internal/plugin/service/organization/billinggroup"
 	"github.com/aiven/terraform-provider-aiven/internal/plugin/service/organization/billinggrouplist"
-	"github.com/aiven/terraform-provider-aiven/internal/plugin/service/organization/org"
+	organization2 "github.com/aiven/terraform-provider-aiven/internal/plugin/service/organization/organization"
```

Comment:
nit: I guess this package will remain as organization, so maybe in makes sense to add an alias for `github.com/aiven/terraform-provider-aiven/internal/plugin/service/organization` instead

Reply:
Will be replaced with the migration of the rest of the resources.